src/Tools/dotnet-monitor/ServiceCollectionExtensions.cs

```diff
@@ -102,6 +107,9 @@ public static IServiceCollection ConfigureCollectionRules(this IServiceCollectio
             services.AddSingleton<ICollectionRuleActionOperations, CollectionRuleActionOperations>();
             services.AddSingleton<ICollectionRuleTriggerOperations, CollectionRuleTriggerOperations>();
 
+            services.AddSingleton<IPostConfigureOptions<TemplateOptions>, TemplatesPostConfigureOptions>();
+            services.AddSingleton<IPostConfigureOptions<CollectionRuleOptions>, CollectionRulePostConfigureOptions>();
```

Comment:
nit: I think it makes sense to move this to between lines 113 and 114 so that all of the `CollectionRuleOptions` options configuration implementations are grouped together. You'd also have to move the `DefaultCollectionRulePostConfigureOptions` registration to this group as well to make this true.